libr/core/cmd_debug.c

```diff
@@ -5238,7 +5238,11 @@ static int cmd_debug(void *data, const char *input) {
 					}
 				}
 				if (stop != -1) {
-					P ("stopreason=%s\n", r_debug_reason_to_string (stop));
+					if (strcmp (r_debug_reason_to_string (core->dbg->reason.type), "signal") == 0) {
+						P ("stopreason=%s\n", r_signal_to_human (core->dbg->reason.signum));
```

Comment:
why repeat the format string if its just one argument that changes? just use the elvis operator to avoid repeating code